examples/I2C_Scan/I2C_Scan.ino

```diff
@@ -10,13 +10,14 @@ extern Adafruit_TestBed TB;
     || defined(ARDUINO_ADAFRUIT_FEATHER_RP2040) \
     || defined(ARDUINO_ADAFRUIT_QTPY_ESP32S2) \
     || defined(ARDUINO_ADAFRUIT_QTPY_ESP32S3_NOPSRAM) \
-    || defined(ARDUINO_ADAFRUIT_QTPY_ESP32_PICO)
+    || defined(ARDUINO_ADAFRUIT_QTPY_ESP32_PICO) \
+    || defined(ARDUINO_SAM_DUE)
```

Comment:
just to check, would you call the primary port Wire1 or Wire?

Reply:
good question. i'm not sure.

does "primary" mean "Wire"? (and "secondary" means "Wire1", etc.)
or does "primary" mean "whatever Wire is on the commonly used pins that pretty much everything connects to by default"?

i was going with the former. there's probably potential confusion with either approach?

Reply:
maybe update the display print to indicate that primare is Wire and secondary is Wire1?

Reply:
love it!